cmake/FindAPEX.cmake

```diff
@@ -0,0 +1,72 @@
+# Copyright (c)      2014 Thomas Heller
+# Copyright (c) 2007-2012 Hartmut Kaiser
+# Copyright (c) 2010-2011 Matt Anderson
+# Copyright (c) 2011      Bryce Lelbach
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if(NOT TARGET pika_internal::apex)
```

Comment:
Note that I'm not using the name `APEX::apex` here. I realized that we probably shouldn't be using namespaces without `pika` in our `FindX.cmake` files, and this is how I'd probably want to have our other `FindX.cmake` files changed as well. Only using `pika(_internal)` for things that we set up means that there's no chance of conflicts from other dependencies setting up targets within the same namespace. Low probability, but I think it's good hygiene.

Reply:
It actually alread has an issue... #9.